avm/ptn/azd/ml-project/tests/e2e/defaults/dependencies.bicep

```diff
@@ -0,0 +1,63 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the AI Studio Hub Resource to create.')
+param hubName string
+
+@description('Required. The name of the key vault to create.')
+param keyVaultName string
+
+@description('Required. The name of the storage account to create.')
+param storageAccountName string
+
+@description('Optional. Tags of the resource.')
+@metadata({
+  example: '''
+  {
+      "key1": "value1"
+      "key2": "value2"
+  }
+  '''
+})
+param tags object?
+
+module hub 'br/public:avm/res/machine-learning-services/workspace:0.7.0' = {
+  name: '${uniqueString(deployment().name, location)}-hub'
+  params: {
+    name: hubName
+    tags: tags
+    location: location
+    sku: 'Basic'
+    kind: 'Hub'
+    associatedKeyVaultResourceId: keyvault.outputs.resourceId
+    associatedStorageAccountResourceId: storageAccount.outputs.resourceId
+  }
+}
+
+module storageAccount 'br/public:avm/res/storage/storage-account:0.9.1' = {
```

Comment:
Also in this case it would help readability to order modules based on their order of deployment (e.g. put storage and kv before hub)